include/ringmesh/basic/common.h

```diff
@@ -235,4 +235,7 @@ namespace RINGMesh
         index_t iter_{ 0 };
         index_t last_{ 0 };
     };
+
+    template < index_t DIMENSION >
+    vecn< DIMENSION > initialize_vecn_coordinates( const double& value );
```

Comment:
The const & is not needed. Passing double or double& cost the same. Maybe removing them for cleaner interface... http://cpp.sh/8rntm

Reply:
Done